enginetest/script_queries.go

```diff
@@ -976,6 +976,14 @@ var ScriptTests = []ScriptTest{
 					{"v1", "bigint", "NO", "", "200", ""},
 				},
 			},
+			{
+				// as of syntax is supported, but mocked out in test harnesses
+				Query: "DESCRIBE test as of 'version'",
```

Comment:
Any reason you can't put this in TestVersionedQueries? The in memory DB has a real (kind of) versioning implementation you could use

Reply:
oh snap! that's a much better place to test at the go-mysql-layer. thanks for pointing that out.